src/pages/ProfilesExplorerView/components/SceneProfilesExplorer/SceneProfilesExplorer.tsx

```diff
@@ -154,13 +155,10 @@ export class SceneProfilesExplorer extends SceneObjectBase<SceneProfilesExplorer
   }
 
   updateFromUrl(values: SceneObjectUrlValues) {
-    const stateUpdate: Partial<SceneProfilesExplorerState> = {};
-
     if (typeof values.explorationType === 'string' && values.explorationType !== this.state.explorationType) {
-      stateUpdate.explorationType = values.explorationType as ExplorationType;
+      const type = values.explorationType as ExplorationType;
+      this.setExplorationType({ type });
```

Comment:
Switching to the getter to get the body updated correctly when going back in history

Reply:
👍🏾 The only drawback is that the same body is built twice - only when landing on the page for the 1st time, if there's an `explorationType` parameter in the URL.